main.py

```diff
@@ -0,0 +1,4 @@
+
+def hello(event, context):
```

Comment:
V simple to verify packaging, deployment and execution works. Actual application code coming in a separate PR